src/main/java/pro/sky/calculator01/service/CalculatorService.java

```diff
@@ -0,0 +1,27 @@
+package pro.sky.calculator01.service;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class CalculatorService {
+    public int resultPlus;
+    public int resultMinus;
+    public int resultMultiplication;
+    public double resultDivision;
+
+    public int plus(int num1, int num2) {
+        return resultPlus = num1 + num2;
+    }
+
+    public int minus(int num1, int num2) {
+        return resultMinus = num1 - num2;
+    }
+
+    public int multiplication(int num1, int num2) {
+        return resultMultiplication = num1 * num2;
```

Comment:
зачем присваивать в поле результат? какой смысл в этом? надо просто вернуть
```suggestion
        return num1 * num2;
```